### ISSUE TITLE
Searching for one Chinese character or for one single-character Chinese word does not work. A minimum of 2 are required.

### ISSUE THREAD

**Issue by SergeantConfused** [bitwarden/web#1419]
### Steps To Reproduce

1. Create a Secure Note vault item with a Chinese Name value, and a Notes field filled with Chinese words.
2. Attempt to search for that vault item by entering a single-character word from its Name value.

### Expected Result

For the search function to display only the matching vault items.
This release: https://bitwarden.com/help/releasenotes/#2021-05-11 included 'Search Internationalization'.

### Actual Result

The search function does nothing; It waits for another character to meet the minimum and start searching.

### Screenshots or Videos

N/A.

### Additional Context

This was tested while the operating system's UI and the Web Vault's UI were set to Chinese (Simplified).

Another behaviour worth noting is that the Search function matches only whole words from the Notes field of vault items. This means that if you have single-character Chinese words without spaces between them in the Notes field of a Secure Note vault item, you would not be able to search for a single word; You'd need to type the entire text for it to match as a "single" word.

### Operating System

Windows

### Operating System Version

Windows 10 Pro.

### Web Browser

Microsoft Edge

### Browser Version

97.0.1072.76.

### Build Version

2.25.1.

**Comment by Greenderella** [comment]
This issue is also present in the mobile application.

**Comment by Hinton** [comment]
From what I can see the behavior hasn't changed since that release, as the code should still be there. But it seems that something is off if it's not working with the locale set to `ch-CN`. Relevant PR: https://github.com/bitwarden/jslib/pull/330

**Comment by eliykat** [comment]
I don't think this behaviour should change based on the locale. We should enable single character searching, but:
* require the user to manually trigger the search, rather than using a debounce
* have some kind of interrupt if it's returning too many results
